src/client/Microsoft.Identity.Client/TelemetryCore/OpenTelemetry/IOpenTelemetryClient.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Identity.Client.Core;
+
+namespace Microsoft.Identity.Client.TelemetryCore.OpenTelemetry
+{
+    internal interface IOpenTelemetryClient
```

Comment:
Is "Client" a bit overloaded word here? Telemetry is more a consumer producer pattern.

Reply:
Updated the name to OTelInstrumentation